radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<li>Automatic ingestion of any log</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Requires some maintenance and consideration on the indices themselves</li>
+<li>Doesn’t have</li>
+</ul>
+<p><a href=""https://engineering.grab.com/structured-logging"">Structured logs</a> should be used where possible to make the searches as efficient as possible.</p>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Kubernetes,Infrastructure,Invest,lock,"<p>Kubernetes is a container orchestration system. Kubernetes has become the de facto standard in the last few years when it comes to running containers.</p>
+<h3>Pros</h3>
+<ul>
+<li>Provides infinite scaling abilities</li>
+<li>Self healing pods when configured with health and readiness probes</li>
+<li>Efficiently packs applications onto the nodes to achieve greater application density which leads to lower costs</li>
+<li>Extremely active community that is constantly contributing back to the ecosystem</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Initial configuration can be very daunting</li>
+<li>Required updates on the regular to not fall behind</li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2022</p>
+"
+Docker,Infrastructure,Invest,lock,"<p>Docker is a means of packaging an application with all of its dependencies required to run. You can think of a Docker container as a lightweight virtual machine without the overhead of the OS. Typically we prefer smaller containers based on the Alpine images. When you build your applications this way you can have the same deployment patterns with all applications regardless of language.</p>
+<h3>Pros</h3>
+<ul>
+<li>Provides a means to control the running environments easily</li>
+<li>The container provides a consistent experience across the board</li>
+<li>Containers enable greater application density to use every ounce of server resources efficiently</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Debugging can be more difficult if there is an issue with the container itself</li>
+</ul>
+<p>Combining these benefits with <a href="""">Kubernetes</a> leads to a great way to manage all applications efficiently.</p>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+AWS Lambda (event processing),Infrastructure,Invest,confident,"<p>AWS Lambda functions are a great way to run snippets of code without worrying about the underlying infrastructure the code would run on. Currently this is most useful as means to cheaply process events from Kinesis, DynamoDb streams, SNS, or SQS. AWS has provided all of the plumbing to handle this efficiently.</p>
+<h3>Pros</h3>
+<ul>
+<li>Very cheap, insanely cheap</li>
+<li>Effectively infinite scale</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Not a good fit for IO bound operations</li>
+<li>Deployment pipelines for the functions themselves is not ideal. Even when using <a href="""">Terraform</a> you have to resort to zip files.</li>
```

Comment:
Missing link for Terraform.  I'm going to stop tagging missing links but there are more

Reply:
Added link

Reply:
The readme states "If you want to link to another description from inside a markdown file you can simply put an empty href like [Angular]()" That doesn't seem to be working in practice.